CRM/Contribute/BAO/ContributionSoft.php

```diff
@@ -607,8 +607,8 @@ protected static function processPCP($pcp, $contribution) {
       $softParams['pcp_personal_note'] = $pcp['pcp_personal_note'] ?? NULL;
       $softParams['soft_credit_type_id'] = CRM_Core_PseudoConstant::getKey('CRM_Contribute_BAO_ContributionSoft', 'soft_credit_type_id', 'pcp');
       $contributionSoft = self::add($softParams);
-      //Send notification to owner for PCP
-      if ($contributionSoft->pcp_id && empty($pcpId)) {
+      // Send notification to owner for PCP.
+      if ($contributionSoft->pcp_id && empty($pcpId) && empty($contribution->contribution_page_id)) {
```

Comment:
@nishant-bhorodia I feel like this might be the wrong extra criteria to add. Does it work if instead we change the contribution status? (if it's still pending we should not send)

Reply:
Hello @eileenmcnaughton 
thanks for reviewing!
I have added another condition to check if the status is `completed`.
The two conditions I added does the following:
1. Check if the contribution is made without contribution page i.e., from the admin area.
2. Check if the status is completed.
Thanks!

Reply:
Hello @eileenmcnaughton ,
Hope you are well!!
Please can I ask if the change looks fine?
thanks!!